www/src/pages/docs/100_Web Components.md

```diff
@@ -79,18 +79,17 @@ object WeatherCard : WebComponent<HTMLDivElement>() {
         }
 }
 ```
-We also add an observed attribute named `city` which then can be set by the user of our component.
-Next, register your component:
+Also add an observed attribute named `city` to be set by the user of this component. Next, register the component and add the observed `city` to the registration.
```

Comment:
Respect the maximum line length please and add some line break. (That might be needed in other places as well!)